packages/syft/src/syft/service/worker/utils.py

```diff
@@ -535,47 +549,20 @@ def create_default_image(
     tag: str,
     in_kubernetes: bool = False,
 ) -> SyftError | SyftWorkerImage:
-    # TODO: Hardcode worker dockerfile since not able to COPY
-    # worker_cpu.dockerfile to backend in backend.dockerfile.
-
-    # default_cpu_dockerfile = get_syft_cpu_dockerfile()
-    # DockerWorkerConfig.from_path(default_cpu_dockerfile)
-
     if not in_kubernetes:
```

Comment:
Kubernetes and Docker are now in parity - no need to build a new worker image - just run the backend in worker mode